### DISCUSSION TITLE
<class 'spatialmath.pose3d.SE3'>

### DISCUSSION THREAD

**Discussion by pinocchio-1** [petercorke/robotics-toolbox-python#313] [Q&A]
Hello，i'm a fresh man in this field. How can I convert <class 'spatialmath.pose3d.SE3'> to 4*4  numpy array （because I want to refer every single element of the fkine result ）.Thank you very much.
For example:
T =  myrobot.fkine(q)
print(T[1][3])-----ERROR

**Comment by micsc12** [comment]
Hello :) 
SE3 objects have a .data property, which you can use. Remember that SE3 can contain both a single pose or a list of poses, which is why it returns a list of numpy arrays. Try this: 

```
# show all poses 
print(T.data)
# show first pose in list
print(T.data[0])
# access translational part of first pose: 
print(T.data[0][0:3,3])
```

**Reply by petercorke** [reply]
The preferred way is `T.A` where `A` stands for (numpy) array.  This is common to all the pose classes.
